clientgui/ViewStatistics.cpp

```diff
@@ -1945,6 +1945,8 @@ CViewStatistics::CViewStatistics(wxNotebook* pNotebook) :
 	m_PaintStatistics = new CPaintStatistics(this, ID_LIST_STATISTICSVIEW, wxDefaultPosition, wxSize(-1, -1), 0);
 	wxASSERT(m_PaintStatistics);
 
+	m_pTaskPane->SetLayoutDirection(wxLayout_LeftToRight);
```

Comment:
Could you please fix indents it this whole funtion?

Reply:
Yes, I did